<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditor</class>
 <widget class="QWidget" name="ImageEditor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑器</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="plusBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 18pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="minusBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="colorBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
